requirements.txt

```diff
@@ -1,3 +1,5 @@
 Flask-SQLAlchemy==2.3.2
-py-ms==0.1.1
-SQLAlchemy==1.2.13
\ No newline at end of file
+# py-ms==0.1.1
+git+https://github.com/python-microservices/pyms.git@master#egg=py-ms
+Flask-Script==2.0.6
+connexion[swagger-ui]
```

Comment:
Esto debería estar en pyms no aquí

Reply:
Si

Reply:
Está fallando al mover conexion[swagger-ui]. Se está investigando, pero el fallo es:
The swagger_ui directory could not be found.
    Please install connexion with extra install: pip install connexion[swagger-ui]
    or provide the path to your local installation by passing swagger_path=<your path>

Reply:
Lo he dejado en esta versión para poder desbloquear y dejar estable la rama master y poder tratarlo en una rama aparte.